src/index.js

```diff
@@ -1,6 +1,17 @@
-const fibonacci = (n) => {
+const fibonacci = (n) =>{ 
+ 
+    let arr=[1];
+    let num1=1;
+    let num=0;
+    for (let index = 0;n>1 && index <= n-2; index++) {
+      newNum=num+num1;
```

Comment:
use `const`: `const newNum = num + num1`. If you don't use `const` or `let`, you don't control scopes.